internal/journal/voljournal.go

```diff
@@ -173,6 +177,7 @@ func NewCSISnapshotJournal(suffix string) *Config {
 		cephSnapSourceKey:       "csi.source",
 		namespace:               "",
 		encryptKMSKey:           "csi.volume.encryptKMS",
+		commonPrefix:            "csi.",
```

Comment:
will this cause any breakage once user upgrades cephcsi, just want to make sure everything works in an upgrade scenario?
the upgrade is not covered in E2E :(

Reply:
It should be fine. It's just a constant that "rolls up" the current practice of all constants that start with "csi." (and I checked, they all do AFAICT).
If this were to change then we'd have to modify it with either additional constants or something, but I don't see a problem here.